uicomponent-compose/other/src/main/java/io/github/droidkaigi/feeder/other/OtherScreen.kt

```diff
@@ -106,7 +109,7 @@ fun OtherScreen(
                     modifier = Modifier.fillMaxHeight()
                 ) {
                     FadeThrough(targetState = selectedTab) { selectedTab ->
-                        BackdropFrontLayerContent(selectedTab, onContributorClick, onStaffClick)
+                        BackdropFrontLayerContent(selectedTab, onContributorClick, onStaffClick, onPrivacyPolicyClick)
```

Comment:
This seems to cause CI to fail, so can I add a newline? 🙏 

```
Error on line: 112, column: 1
  Exceeded max line length (100)
```

Reply:
Sorry! I'll fix it! 🙇